pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -1688,13 +1688,13 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					Label:        "Subject",
 					Element:      ElementTypeTextArea,
 					InputType:    InputTypeText,
-					Description:  "Optional subject. You can use templates to customize this field",
+					Description:  "Optional subject. By default, it uses the default title template and can be customized with templates and custom messages. It cannot be an empty string",
```

Comment:
```suggestion
					Description:  "Optional subject. By default, this field uses the default title template and can be customized with templates and custom messages. It cannot be an empty string",
```
minor suggestion: the word "it" should be predicated with a more specific noun.